src/test/ui/rfc-2361-dbg-macro/dbg-macro-move-semantics.nll.stderr

```diff
@@ -0,0 +1,36 @@
+error[E0382]: use of moved value: `a`
+  --> $DIR/dbg-macro-move-semantics.rs:11:18
+   |
+LL |     let _ = dbg!(a);
+   |             ------- value moved here
+LL |     let _ = dbg!(a);
+   |                  ^ value used here after move
+   |
+   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
+   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
+
+error[E0382]: borrow of moved value: `a`
+  --> $DIR/dbg-macro-move-semantics.rs:11:18
+   |
+LL |     let _ = dbg!(a);
+   |             ------- value moved here
+LL |     let _ = dbg!(a);
+   |                  ^ value borrowed here after move
+   |
+   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
+   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
+
+error[E0382]: use of moved value: `a`
+  --> $DIR/dbg-macro-move-semantics.rs:11:13
+   |
+LL |     let _ = dbg!(a);
+   |             ------- value moved here
+LL |     let _ = dbg!(a);
+   |             ^^^^^^^ value used here after move
+   |
+   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
+   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
+
+error: aborting due to 3 previous errors
```

Comment:
```
[01:03:52] ---- [ui (nll)] ui/rfc-2361-dbg-macro/dbg-macro-move-semantics.rs stdout ----
[01:03:52] diff of stderr:
[01:03:52] 
[01:03:52] 9	   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
[01:03:52] 10	   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
[01:03:52] 11	
[01:03:52] -	error[E0382]: borrow of moved value: `a`
[01:03:52] -	  --> $DIR/dbg-macro-move-semantics.rs:11:18
[01:03:52] -	   |
[01:03:52] -	LL |     let _ = dbg!(a);
[01:03:52] -	   |             ------- value moved here
[01:03:52] -	LL |     let _ = dbg!(a);
[01:03:52] -	   |                  ^ value borrowed here after move
[01:03:52] -	   |
[01:03:52] -	   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
[01:03:52] -	   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
[01:03:52] -	
[01:03:52] -	error[E0382]: use of moved value: `a`
[01:03:52] -	  --> $DIR/dbg-macro-move-semantics.rs:11:13
[01:03:52] -	   |
[01:03:52] -	LL |     let _ = dbg!(a);
[01:03:52] -	   |             ------- value moved here
[01:03:52] -	LL |     let _ = dbg!(a);
[01:03:52] -	   |             ^^^^^^^ value used here after move
[01:03:52] -	   |
[01:03:52] -	   = note: move occurs because `a` has type `NoCopy`, which does not implement the `Copy` trait
[01:03:52] -	   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
[01:03:52] -	
[01:03:52] -	error: aborting due to 3 previous errors
[01:03:52] +	error: aborting due to previous error
[01:03:52] 35	
[01:03:52] 36	For more information about this error, try `rustc --explain E0382`.
[01:03:52] 37	
```